client/src/styles/app.css

```diff
@@ -24,10 +24,15 @@ body{
 }
 
 .btn-restaurants{
-  font-size: 150%;
+  font-size: 160%;
 }
 
 .btn-restaurants:hover{
```

Comment:
![image](https://user-images.githubusercontent.com/6152066/80124250-a819ae80-8598-11ea-9aba-45d2865099c9.png)

Button is almost invisible over this background